.github/workflows/python-cicd.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: TFRecord CICD
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
```

Comment:
We are supporting Python 3.6+ in our PRD, so maybe we can set this on the lower bound.

Reply:
Can try it out and make sure it works.  AFAIK we haven't tested 3.6 so far.